nominees/coinsence.json

```diff
@@ -1,16 +1,31 @@
 {
   "name": "Coinsence",
-  "description": "Platform to match youth volunteers to social enterprises",
+  "aliases": [
+    ""
+  ],
+  "description": "A global decentral collaboration platform and change-maker network powered by blockchain based impact currencies",
   "website": "https://coinsence.org",
   "license": [
     {
       "spdx": "MIT",
-      "licenseURL": "https://opensource.org/licenses/mit-license.php"
+      "licenseURL": "https://github.com/Coinsence/humhub-modules-xcoin/blob/master/LICENSE"
```

Comment:
Are they only certifying their Xcoin Module? If so, then I think the title of this submission should be changed, e.g. `Coinsense Xcoin Module` instead of `Coinsense`. That seems to include the larger project, which is not submitted here.